src/main/java/com/minecolonies/coremod/colony/TriggerColonyAchievements.java

```diff
@@ -0,0 +1,232 @@
+package com.minecolonies.coremod.colony;
+
+import com.minecolonies.coremod.achievements.ModAchievements;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Trigger the corresponding colony achievement.
+ */
+public class TriggerColonyAchievements
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1118)